hipogram/posts/views.py

```diff
@@ -17,6 +17,13 @@ class PostListView(ListView):
     template_name = "post_list.html"
     paginate_by = 2
 
+    def get_queryset(self):
+        queryset = Post.objects.all()
+        name = self.request.GET.get('name')
+        if name:
```

Comment:
I like to use [walrus operator](https://docs.python.org/3/whatsnew/3.8.html) in this type of situations. This block could be like this:

```python
def get_queryset(self):
    queryset = Post.objects.all()
    if name := self.request.GET.get('name'):
        queryset.filter(created_by=name)
    return queryset
```

This is not mandatory of course, I just mention that something I like.